code/Gamemodes/Modes/FFA/FreeForAll.cs

```diff
@@ -111,8 +111,8 @@ private void ZoneTrigger()
 			}
 		}
 
-		var projectiles = All.OfType<Projectile>();
-		foreach ( var proj in projectiles )
+		var explosives = All.OfType<Explosive>();

```

Comment:
So I understand that projectiles are essentially just explosives now, but I think we should give Explosive a parent class of Projectile that we can extend the behaviour of. That way, when we add in projectiles that don't necessarily explode, we can still simulate it all easily within this list, and the camera behaves correctly, etc.